packages/fe-container/src/components/dark-mode-switch/DarkModeSwitch.tsx

```diff
@@ -1,24 +1,31 @@
 import React, {useCallback, useState} from 'react'
 import {FormattedMessage} from 'react-intl'
+import PropTypes from 'prop-types'
 
 import {switchTheme} from '@utils/theme/ThemeManager'
 
 import './DarkModeSwitch.less'
 
-export const DarkModeSwitch: React.FC = () => {
+const darkModeSwitchProps = {
+  toggleCallback: PropTypes.func.isRequired
+}
+
+type DarkModeSwitchProps = PropTypes.InferProps<typeof darkModeSwitchProps>
```

Comment:
Because you use this for both components, you can consider to call it `SwitchProps`